docs/book/v3/features/error-handling.md

```diff
@@ -213,6 +213,23 @@ class LoggingErrorListenerDelegatorFactory
 }
 ```
 
+Then, enable the delegator factory by registering it in the `delegators` element of the relevant [ConfigProvider class](https://docs.laminas.dev/laminas-config-aggregator/config-providers/).
+For example, in _src/App/src/ConfigProvider.php_, update the `getDependencies()` method as follows:
```

Comment:
```suggestion
For example, in `src/App/src/ConfigProvider.php`, update the `getDependencies()` method as follows:
```

Reply:
As mentioned in https://github.com/mezzio/mezzio/pull/116#discussion_r893430990, I italicised the path as I picked up that convention while working at Twilio. It nicely separates files from code, which I hope makes the docs that much easier to scan/read. Would you mind me making this formatting change?

Reply:
While I love the italicised approach, I accept your wisdom.